lib/stub_cas.rb

```diff
@@ -0,0 +1,26 @@
+require 'webmock'
+
+# Stub out HTTP requests to the remote CAS service, for testing / development.
+# If an email address is provided in the request (the typical cas), we return
+# that email address as the user's GUID.
+class StubCas
```

Comment:
I normally would expect this sort of file to go in the test/support directory, unless it's needed outside of tests.